PyDSS/pydss_project.py

```diff
@@ -585,11 +604,43 @@ def serialize(self, path):
                 os.path.join(path, "ExportLists", filename_from_enum(mode))
             )
 
+        for visualization_type, visualizations in self.visualizations.items():
+            filename = os.path.join(
+                path, "pyPlotList", filename_from_enum(visualization_type)
+            )
+            dump_data(visualizations, filename)
+
+        # @Danial the plots.toml file is not used by a single scenario.
```

Comment:
OK.  I don't remember why I thought it was scenario-specific.

Reply:
taken care of